docker-hub/image-management.md

```diff
@@ -4,69 +4,34 @@ keywords: dashboard, images, image management, inactive
 title: Advanced Image Management dashboard
 ---
 
-{% include upgrade-cta.html
-  body="Docker Pro, Team, and Business users can now use Docker Hub's **Advanced Image Management** dashboard. The dashboard enables a new level of access to your content, allowing fine-grained pruning of old data and exploration of old versions of pushed images. Upgrade now to start using the Advanced Image Management dashboard."
-  header-text="This feature requires a paid Docker subscription"
-  target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_image-mgmt"
-%}
+{% include upgrade-cta.html %}
```

Comment:
Not sure if intentional. There is no more text in the box. Just the upgrade button.

Reply:
Yes intentional as a lot of the body text just repeats what is said in the main body of the text and just made it look confusing